library/src/main/java/com/novoda/downloadmanager/lib/DatabaseHelper.java

```diff
@@ -151,28 +151,28 @@ private void createDownloadsByBatchView(SQLiteDatabase db) {
      * columns to request from DownloadProvider.
      */
     public static final String[] DOWNLOAD_BY_BATCH_VIEW_COLUMNS = new String[]{
-            Downloads.Impl.DOWNLOADS_TABLE_NAME + "." + Downloads.Impl._ID + " AS _id ",
-            Downloads.Impl._DATA,
-            Downloads.Impl.COLUMN_MEDIAPROVIDER_URI,
-            Downloads.Impl.COLUMN_DESTINATION,
-            Downloads.Impl.COLUMN_URI,
-            Downloads.Impl.COLUMN_STATUS,
-            Downloads.Impl.DOWNLOADS_TABLE_NAME + "." + Downloads.Impl.COLUMN_DELETED,
-            Downloads.Impl.COLUMN_FILE_NAME_HINT,
-            Downloads.Impl.COLUMN_MIME_TYPE,
-            Downloads.Impl.COLUMN_TOTAL_BYTES,
-            Downloads.Impl.COLUMN_LAST_MODIFICATION,
-            Downloads.Impl.COLUMN_CURRENT_BYTES,
-            Downloads.Impl.COLUMN_NOTIFICATION_EXTRAS,
-            Downloads.Impl.COLUMN_BATCH_ID,
-            Downloads.Impl.Batches.COLUMN_TITLE,
-            Downloads.Impl.Batches.COLUMN_DESCRIPTION,
-            Downloads.Impl.Batches.COLUMN_BIG_PICTURE,
-            Downloads.Impl.Batches.COLUMN_VISIBILITY,
-            Downloads.Impl.Batches.COLUMN_STATUS,
-            Downloads.Impl.Batches.BATCHES_TABLE_NAME + "." + Downloads.Impl.Batches.COLUMN_DELETED,
-            Downloads.Impl.Batches.COLUMN_TOTAL_BYTES,
-            Downloads.Impl.Batches.COLUMN_CURRENT_BYTES
+            DownloadsContract.DOWNLOADS_TABLE_NAME + "." + DownloadsContract._ID + " AS _id ",
+            DownloadsContract.COLUMN_DATA,
+            DownloadsContract.COLUMN_MEDIAPROVIDER_URI,
+            DownloadsContract.COLUMN_DESTINATION,
+            DownloadsContract.COLUMN_URI,
+            DownloadsContract.COLUMN_STATUS,
+            DownloadsContract.DOWNLOADS_TABLE_NAME + "." + DownloadsContract.COLUMN_DELETED,
+            DownloadsContract.COLUMN_FILE_NAME_HINT,
+            DownloadsContract.COLUMN_MIME_TYPE,
+            DownloadsContract.COLUMN_TOTAL_BYTES,
+            DownloadsContract.COLUMN_LAST_MODIFICATION,
+            DownloadsContract.COLUMN_CURRENT_BYTES,
+            DownloadsContract.COLUMN_NOTIFICATION_EXTRAS,
+            DownloadsContract.COLUMN_BATCH_ID,
+            BatchesContract.COLUMN_TITLE,
```

Comment:
yeah i wrote it on the other comment. The contract is a database contract then each class represents a table or a view, yes we'll have a giant class but i think this is a good hting